databases/week1/week1.sql

```diff
@@ -0,0 +1,52 @@
+#1
+SELECT COUNT(*) AS number_of_tasks
+FROM task;
+
+#2
+SELECT COUNT(*)
+FROM task 
+WHERE  due_date IS NULL;
+
+#3
+SELECT * FROM task 
```

Comment:
Avoid using * everywhere. Use it only if necessary. For example, here you can make a few specific columns selection as 
```SELECT id,  title, description 
FROM task
JOIN status on task.status_id=status.id
 WHERE name = "Done"; ```

This applies until Q # 7